### ISSUE TITLE
Headers: Styling and WCAG

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#35]
## Description

As an app-developer I want my application to be WCAG compatible and the visual representation that you expect with the Header tag. This issue was raised because the default value is h4. Headers can not jump over a header level in descending order. [Tilsynet](https://www.uutilsynet.no/wcag-standarden/tekst-og-struktur/226#overskrifter)

## Specification tasks
- [x] Write and update guidelines for the header component
- [x] Decide what the default value should be
- [ ] Check if headers have the same styling in production and in Figma

**Comment by Febakke** [comment]
@mrosvik 
Kjøre en siste test på om det er riktige tekstressurser i Figma og i appene. Om det er det kan vi lukke denne.

**Comment by ivarne** [comment]
Må ikke denne fikses med å gi (feil)melding til apputviklere hvis de lager et skjema som ikke følger wcag krav om at overskriftsnivå h1-h5 skal være et sammenhengende tre som egner seg som innholdsfortegnelse over siden? Hva har det med Figma å gjøre?

**Comment by Febakke** [comment]
Jeg så et åpent punkt som gikk på å se på om det er en forskjell på header størrelser vi bruker i vår offentlige Figma fil som de som er i produksjon.

@ivarne 
Absolutt noe vi burde ha på plass i Studio, dette issuet er jo ganske vagt hvis det er det som skal produseres her. Det er for så vidt ikke noe som skal gjøre på front-end heller. Jeg flytter det dermed tilbake til Studio repoet og legger til litt mer utfyllende informasjon.
